src/testFetch.tsx

```diff
@@ -0,0 +1,17 @@
+export async function fetchData(url: string): Promise<any> {
+  const res = await fetch(url);
+
+  if (!res.ok) {
+    throw new Error(`${res.status} ${res.statusText}`);
+  }
+  const contentType = res.headers.get("Content-Type");
+  if (contentType && contentType.indexOf("application/json") !== -1) {
+    console.log(res.json());
```

Comment:
```suggestion
```
I'm getting this error: Failed to execute 'json' on 'Response': body stream already read.
-> Delete this line, as the res gets already modified by it. The subsequent res operation then throws an error.